### PR TITLE
Fix unit test sleeps and unit-test config check synchronously

### PR DESCRIPTION
**Pull request by loffek**
Ignore whitespace changes ;)

### CODE REVIEW

File: src/main/java/de/azapps/kafkabackup/common/topic/TopicConfigurationService.java [hunk 1]
```diff
@@ -27,51 +27,51 @@
   Semaphore configurationCheckSemaphore = new Semaphore(1);
 
   public void runTopicConfigurationCheck() {
```
Comment (MacRudDNA) [inline src/main/java/de/azapps/kafkabackup/common/topic/TopicConfigurationService.java]: Let's move this asynchronous method to BackupSinkTask class. This way service will be synchronous and in sink task we decide to make it asynchronous.
WDYT?

**Review by MacRudDNA — COMMENTED** [review]
One suggestion. Otherwise looks good to me.

File: src/main/java/de/azapps/kafkabackup/common/topic/TopicConfigurationService.java (hunk 1 above)
Comment (loffek) [reply src/main/java/de/azapps/kafkabackup/common/topic/TopicConfigurationService.java]: Very nice! Will do.

**Review by MacRudDNA — APPROVED** [review]
LGTM 👍

**Pull request merged**